bedrock/firefox/views.py

```diff
@@ -379,7 +379,6 @@ class WhatsnewView(L10nTemplateView):
         "firefox/developer/whatsnew.html": ["firefox/developer"],
         "firefox/developer/whatsnew-mdnplus.html": ["firefox/whatsnew/whatsnew-developer-mdnplus"],
         "firefox/nightly/whatsnew.html": ["firefox/nightly/whatsnew", "firefox/whatsnew/whatsnew"],
-        "firefox/whatsnew/index-account.html": ["firefox/whatsnew/whatsnew-account", "firefox/whatsnew/whatsnew"],
```

Comment:
**Question:** Can we remove `firefox/whatsnew/whatsnew-account.ftl`? And also from [pontoon.toml](https://github.com/mozilla/bedrock/blob/main/l10n/configs/pontoon.toml#L194-L196)?